sdk/core/core-http-compat/review/core-http-compat.api.md

```diff
@@ -0,0 +1,44 @@
+## API Report File for "@azure/core-http-compat"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { CommonClientOptions } from '@azure/core-client';
+import { ServiceClient } from '@azure/core-client';
+import { ServiceClientOptions } from '@azure/core-client';
+
+// @public (undocumented)
+export interface KeepAliveOptions {
+    enable?: boolean;
+}
+
+// @public (undocumented)
+export interface RedirectPolicyOptions {
+    // (undocumented)
+    handleRedirects?: boolean;
+    maxRetries?: number;
+}
+
+// @public (undocumented)
+export class ShimClient extends ServiceClient {
```

Comment:
Since our ref docs will show the clients in our packages extend ShimClient, can we have a better name for it? How about `ExtendedServiceClient`?

Reply:
`LegacyServiceClient`?

Reply:
I would be worried if a package I am using has classes extending from "legacy" things :(

Reply:
I have renamed it to ExtendedServiceClient for now